invenio_administration/translations/de/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,123 @@
+# German translations for invenio-administration.
+# Copyright (C) 2022 CERN
+# This file is distributed under the same license as the
+# invenio-administration project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: invenio-administration 0.1.0\n"
+"Report-Msgid-Bugs-To: info@inveniosoftware.org\n"
+"POT-Creation-Date: 2022-09-14 16:18+0200\n"
+"PO-Revision-Date: 2022-09-14 16:19+0200\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: de\n"
+"Language-Team: de <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.9.1\n"
+
+#: invenio_administration/errors.py:21
+msgid "Invalid resource {resource} configured for view {view}."
```

Comment:
how was this file generated?

Reply:
The translation keys were extracted as explained in the doc https://inveniordm.docs.cern.ch/develop/best-practices/i18n/#python_2. The only difference is that the use-fuzzy variable was not set, as in the other repositories.